.github/workflows/docs_build_and_deploy.yml

```diff
@@ -27,9 +27,48 @@ jobs:
     name: Build Sphinx Docs
     runs-on: ubuntu-latest
     steps:
-      - uses: neuroinformatics-unit/actions/build_sphinx_docs@main
-        with:
-          python-version: 3.11
+    - uses: actions/checkout@v4
```

Comment:
Should we define this action in https://github.com/neuroinformatics-unit/actions ?

It seems like it could be useful for other repos.
(Or is it not generalisable because it assumes specific settings in the Make files?)

Reply:
good call. i'll make a PR there for this.

Reply:
[PR](https://github.com/neuroinformatics-unit/actions/pull/58) opened in [actions]( https://github.com/neuroinformatics-unit/actions)